src/style.css

```diff
@@ -1,3 +1,71 @@
+html {
+  box-sizing: border-box;
```

Comment:
The `*` selector is generally not recommended as it can lead to performance issues. It applies styles to every single element on the page, which can be inefficient. Instead, try to target specific elements that need the styles.